src/main.js

```diff
@@ -35,60 +37,77 @@ randomButton.addEventListener('click', function() {
 formButton.addEventListener('click', switchToFormView);
 viewSavedButton.addEventListener('click', switchToSavedView);
 homeButton.addEventListener('click', switchToHomeView);
+saveButton.addEventListener('click', saveVisibleCover)
 
-//event handlers
+// **** view switch functions ****
 function switchToFormView() {
-  homeView.classList.add('hidden'); //hide home view
-  savedView.classList.add('hidden'); //hide saved view
-  formView.classList.remove('hidden'); //reveal form view
-  homeButton.classList.remove('hidden'); //reveal home button
-  randomButton.classList.add('hidden'); //hide random cover button
-  saveButton.classList.add('hidden'); //hide saved cover button
+  homeView.classList.add("hidden"); //hide home view
+  savedView.classList.add("hidden"); //hide saved view
+  formView.classList.remove("hidden"); //reveal form view
+  homeButton.classList.remove("hidden"); //reveal home button
+  randomButton.classList.add("hidden"); //hide random cover button
+  saveButton.classList.add("hidden"); //hide saved cover button
```

Comment:
There are a lot of these. You don't have to re-add hidden classes in order for them to stay hidden. I'd also check out `classList.toggle()` because there's a big opportunity to DRY up your code here.